spec/helpers/feedbacks_helper_spec.rb

```diff
@@ -29,6 +29,17 @@ class Klass
         klass.open_requests(feedback.user).should == 1
       end
     end
+
+    describe 'invitations and submitted feedback' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.